src/audio-worklet/free-queue/example/README.md

```diff
@@ -0,0 +1,44 @@
+# Free Queue C
+
+Example of Using FreeQueue with WebAssembly App written in C.
+
+
+
+## Building
+
+Install and Activate Emscripten SDK and use
+
+`build.sh` for Linux 
```

Comment:
What about MacOS?

Reply:
Actually, I don't have access to mac, nor experience about how to write scripts for it 😅